### PR TITLE
Fixed stack echo chara_1 not showing up in the mini-preview

### PR DESCRIPTION
**Pull request by Coolsonickirby**
- Fixed stack echo chara_1 not showing up in the mini-preview

### CODE REVIEW

File: src/fixes.rs [hunk 1]
```diff
@@ -104,29 +109,21 @@ fn install_lazy_loading_patches(){
...
         // If we have the first and fourth param in our cache, then we're in the
         // character select screen and can load the files manually
         if PARAM_1 != 0 && PARAM_4 != 0 {
             // Get the color_num for smooth loading between different CSPs
             let max_color: u32 = {
                 // Get the character database for the color num function
-                let character_database = {
+                let character_database ={
                     let databases = *((*PARAMATERS_CACHE + 0x8) as *const u64);
                     *((databases + 360) as *const u64) as u64
```
Comment (Raytwo) [inline src/fixes.rs:121]: Make a proper struct so we can expand on it as needed in the future and it is clearer (but feel free to use a array of u64 to skip all the bytes we don't use yet)

File: src/fixes.rs [hunk 2]
```diff
@@ -150,13 +147,29 @@ fn install_lazy_loading_patches(){
...
+        param_1: *const u64,
+        chara_hash_1: u64,
+        chara_hash_2: u64,
+        color: u32,
+        unk1: u32,
+        unk2: u32,
+    ){
+        let character_database = {
+            let databases = *((*PARAMATERS_CACHE + 0x8) as *const u64);
+            *((databases + 360) as *const u64) as u64
```
Comment (Raytwo) [inline src/fixes.rs:168]: Same as previous comment

**Review by Raytwo — CHANGES_REQUESTED** [review]

File: src/fixes.rs (hunk 1 above)
Comment (Coolsonickirby) [reply src/fixes.rs]: done

File: src/fixes.rs (hunk 2 above)
Comment (Coolsonickirby) [reply src/fixes.rs]: done

**Review by Raytwo — APPROVED** [review]

**Pull request merged**